backend/src/main/java/pl/edu/agh/awiteks_backend/models/Species.java

```diff
@@ -10,7 +10,7 @@
 @Table(name = "species")
 @JsonIgnoreProperties({"plantList"})
 public class Species {
-    private static final int NO_CREATOR = -1;
+    public static final int NO_CREATOR = -1;
```

Comment:
dlaczego rezygnujemy z private?

Reply:
Bo jest to używane w metodzie `getAll` w `SpeciesService`, żeby pobrać tylko gatunku ogólnodostępne i użytkownika